src/config.rs

```diff
@@ -59,6 +61,41 @@ pub struct TelemetryConfig {
     pub service_name: String,
 }
 
+/// All the possible countries in which the application can run.
+#[derive(PartialEq, Eq, Debug, Clone)]
+pub enum Country {
+    Common,
+    It,
+    Es,
+    Uk,
+}
+
+impl FromStr for Country {
+    type Err = CountryParseError;
+
+    fn from_str(s: &str) -> Result<Self, Self::Err> {
+        match s {
+            "common" => Ok(Self::Common),
```

Comment:
I think that common doesn't exist anymore.. I think we can remove it

Reply:
My understanding is that it would become a thing now, eg. with us having a single paperboy instance

Reply:
We could ask someone from @primait/devops

Reply:
I'll remove it then, thanks!